schema/properties_directory/structural/basis/atomic_coordinate.json

```diff
@@ -10,10 +9,7 @@
             ]
         },
         "values": {
-            "$ref": "../../../core/reusable/atomic_vectors.json"
-        },
-        "units": {
-            "$ref": "../../../definitions/units.json#/length"
```

Comment:
This all makes sense - no need to remove the `atomic_coordinates` property

Reply:
The problem was in the inconsistent use of the standalone `atomic_coordinates` property inside the `basis` of material.